ZIGSIMPlus/Models/NetworkAdapter.swift

```diff
@@ -67,8 +73,8 @@ public class NetworkAdapter {
             switch tcpClient.connect(timeout: 1) {
             case .success:
                 break
-            case .failure(let e):
-                error = e
+            case let .failure(err):
+                error = err
```

Comment:
You can allow short name for local variables.
But if you want to follow the strict rule, it's OK.

Reply:
yea I wanted to leave the rules default as long as I can.